tests/test_command_line.py

```diff
@@ -131,7 +131,7 @@ def teardown(self):
 
     def test_startup(self):
         # Make sure debug server starts without error
-        port = get_config().get('port')
         p = pexpect.spawn('dallinger', ['debug'])
-        p.expect_exact('Server is running on 0.0.0.0:{}. Press Ctrl+C to exit.'.format(port))
+        p.expect_exact('Server is running')
         p.sendcontrol('c')
+        p.read()
```

Comment:
This was "fun" to track down. Without the read, the pexpect.spawn instance was getting garbage collected and terminating the subprocess somehow before its `atexit` handlers had a chance to run and record coverage data.